### DISCUSSION TITLE
VS 2022 17.8.5 cannot update copilot from 1.137 to 1.149

### DISCUSSION THREAD

**Discussion by glass-edge** [community/community#101594] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

VS 2022 version 17.8.5
GitHub copilot installed version 1.137
Update version 1.149 available in extensions, when trying to install the update, operation fails with message:
"This extension is already installed to all applicable products."

Same message if I try to install GitHub.Copilot.Vsix.1.149.0.0.vsix downloaded separately.

See the attached video for details.

https://github.com/community/community/assets/136560194/1a59f24a-ff40-4fdf-a0da-81b4fd88ed39

**Comment by lolo-x** [comment]
I have exactly the same issue.

**Comment by ayim** [comment]
@lolo-x @glass-edge oh my, not the experience we want you to have (since we defs want you to have the latest bits).
Mind filing a bug using VS Feedback? That'll help us collect some logs and get started on this.
![image](https://github.com/community/community/assets/10951869/a3004e03-a1cb-4822-a6e4-8c17ea6742f1)

-- 
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
github: ayim | aayim@microsoft.com

**Reply by lolo-x** [reply]
Thank you for your reply, it is greatly appreciated.

After experiencing the issue described above for quite some time, GitHub Copilote extension was finally automatically updated to 1.149.0.0. It happened just after GitHub Copilot Chat extension was manually installed. Not sure whether the two things are related.

So, in my case, the issue is resolved - as least for now...

However, please note that the community has been reporting similar issues for a while on various platforms. For example: https://developercommunity.visualstudio.com/t/The-new-GitHub-Copilot-component-cannot/10518211?sort=newest

**Reply by glass-edge** [reply, marked as answer]
So apparently the fix will be coming in VS 2022 17.9. I will wait until it is released. Thanks!

**Reply by daiplusplus** [reply]
@ayim I'm on VS2022 LTSC 17.8, so us LTSC users won't be getting 17.9, will we have to wait until 17.10 in 2024H2 or will the patch be backported?
